armi/tests/test_mpiActions.py

```diff
@@ -114,7 +111,8 @@ def test_distributeSettings(self):
                         del original[key]
                     if key in current:
                         del current[key]
-                for key, val in original.items():
+
+                for key in original.keys():
```

Comment:
`pylint` might even complain that you could just iterate the dict without `.keys()`, but I find `.keys()` more readable. If it is complaining, should we consider silencing the suggestion?

Reply:
Yeah, me putting `keys()` in there was definitely a style choice. And not one I'll fight over for a second, just a choice.

The `pylint` error here was actually that `val` was unnecessary.